fields/types/list/ListType.js

```diff
@@ -185,17 +185,19 @@ list.prototype.updateItem = function (item, data, files, callback) {
 
 	var field = this;
 	var values = this.getValueFromData(data);
-	// Don't update the value when it is undefined
-	if (values === undefined) {
-		return utils.defer(callback);
-	}
-	// Reset the value when null or an empty string is provided
-	if (values === null || values === '') {
+	// Reset the value when no value or null or an empty string is provided
+	if (values === undefined || values === null || values === '') {
```

Comment:
This is a definite change in semantics, I think it would be better to leave the `undefined` case as-is…